src/app/SubscriptionResumptionSessionEstablisher.cpp

```diff
@@ -90,15 +90,21 @@ void SubscriptionResumptionSessionEstablisher::HandleDeviceConnected(void * cont
     AutoDeleteEstablisher establisher(static_cast<SubscriptionResumptionSessionEstablisher *>(context));
     SubscriptionResumptionStorage::SubscriptionInfo & subscriptionInfo = establisher->mSubscriptionInfo;
     InteractionModelEngine * imEngine                                  = InteractionModelEngine::GetInstance();
+
+    // Decrement the number of subscriptions to resume
```

Comment:
Document why, not what.

This comment would be easier to write if we had a clear notion of what the counter is counting.

Reply:
done